<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup</class>
 <widget class="QWidget" name="signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="rbtn_org_p">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>781</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe UI Historic&quot;;
background-color: #4D6D9A;
color:#00007f ;</string>
   </property>
   <property name="title">
    <string>signup</string>
   </property>
   <widget class="QLabel" name="lbl_">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_username_s">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#99CED3 ;
color: rgb(0, 72, 106) ;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_password_s">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>200</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#99CED3 ;
color: rgb(0, 72, 106) ;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_done_signup">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>260</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: #333;
border: 2px solid #555;
border-radius: 20px;
border-style: outset;
background:  #EDB5BF;
padding: 5px;
}

QPushButton:hover {
background:   #ffdcee;
}</string>
    </property>
    <property name="text">
     <string>done</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtn_personal_s">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>60</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Historic</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Personal</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtn_organ_s">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Historic</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>organisation</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtn_ano_s">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Historic</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Anonymous</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_age_s">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Birthday</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_name_s">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_name_s">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#99CED3 ;
color: rgb(0, 72, 106) ;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_phone_s">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Phone Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_phone_s">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#99CED3 ;
color: rgb(0, 72, 106) ;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Country</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_country_s">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#99CED3 ;
color: rgb(0, 72, 106) ;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_bio_s">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>55</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Bio</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_age_s">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#99CED3 ;
color: rgb(0, 72, 106) ;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_link_s">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>55</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Link</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_link_s">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>420</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#99CED3 ;
color: rgb(0, 72, 106) ;</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTxt_bio_s">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>340</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#99CED3 ;
color: rgb(0, 72, 106) ;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_type_s">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Historic</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Choose a type:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
